otherlibs/unix/gethost.c

```diff
@@ -101,15 +101,15 @@ CAMLprim value caml_unix_gethostbyaddr(value a)
 #if HAS_IPV6
   }
 #endif
-#if HAS_GETHOSTBYADDR_R == 7
+#if defined(HAS_GETHOSTBYADDR_R) && HAS_GETHOSTBYADDR_R == 7
```

Comment:
This could be tidied slightly differently:
```c
#if !defined(HAS_GETHOSTBYADDR_R)
/* #else block calling gethostbyaddr here */
#elif HAS_GETHOSTBYADDR_R == 7
/* ... */
```

While moving that `#else` block, one could also remove `GETHOSTBYADDR_IS_REENTRANT` as since #1368 the distinction is irrelevant, and `caml_enter_blocking_section` could simply be guarded by `#ifdef _WIN32`.

Reply:
Much neater, thanks for the suggestion.